sandy/core/cov.py

```diff
@@ -324,12 +324,24 @@ def eig(self, sort=True):
                     0            1
         0 7.07107e-01 -7.07107e-01
         1 7.07107e-01  7.07107e-01
-        
-        >>> endf6 = sandy.get_endf6_file('jeff_33','xs', 922350)
-        >>> Cov = sandy.XsCov.from_endf6(endf6)
+
+        >>> Cov = sandy.random_cov(50,seed=11)
         >>> Cov = sandy.CategoryCov(Cov)
         >>> len(Cov.eig()[0])
-        295
+        50
+
+        >>> Cov = sandy.random_cov(3,seed=1)
+        >>> sandy.CategoryCov(Cov).eig()[0]
+        0    8.49942e-01
+        1    1.18850e-01
+        2   -3.32188e-02

```

Comment:
I am troubled by this negative eigenvalue...is it correct? Is the `random_cov` method correct?